cfgov/v1/models/sublanding_page.py

```diff
@@ -85,11 +85,11 @@ def get_browsefilterable_posts(self, limit):
                         and 'archive' not in p.title.lower()]
         posts_list = []
         for page in filter_pages:
-            base_query = AbstractFilterPage.objects.live().filter(
+            eligible_children = AbstractFilterPage.objects.live().filter(
```

Comment:
Is there a reason this variable is called `eligible_children` as opposed to `filterable_pages`, which it's later passed in as?

Reply:
@richaagarwal no reason; I thought it was a bit more explanatory, as these pages are the children of this page that are eligible to be included in the list of posts.